aries-site/src/examples/templates/FormChildObject/FormChildObject.js

```diff
@@ -0,0 +1,137 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Collapsible,
+  Footer,
+  Heading,
+  Header,
+  Text,
+} from 'grommet';
+import { FormUp, FormDown } from 'grommet-icons';
+
+const getSummaryString = (values, keys) => {
+  let summary = '';
+  Object.entries(values).forEach(([key, value]) => {
+    summary +=
+      keys.includes(key) && (value.length > 0 || typeof value === 'number')
+        ? `${key}: ${value}, `
+        : '';
+  });
+  summary = summary.slice(0, -2);
+  return summary;
+};
+
+export const FormChildObject = ({
+  children,
+  collectionName,
+  index,
+  level,
+  name,
+  onRemove,
+  open: openProp = false,
+  summarize,
+  values,
+}) => {
+  const [open, setOpen] = useState(openProp);
+  const [background, setBackground] = useState(null);
+
+  const borderStyle = { side: 'top', color: 'border-weak' };
+
+  const valuesSummary = summarize ? getSummaryString(values, summarize) : null;
+
+  return (
+    <Box border={borderStyle}>
+      <Button onClick={() => setOpen(!open)}>
+        <ChildHeader
+          background={background}
+          collectionName={collectionName}
+          height={{ min: 'xxsmall' }}
+          index={index}
+          level={level}
+          name={name}
+          open={open}
+          summary={valuesSummary}
+          onMouseEnter={() => setBackground('background-contrast')}
+          onMouseLeave={() => setBackground(null)}
+          pad={{
+            horizontal: 'small',
+            vertical: 'small',
+          }}
+        />
+      </Button>
+      <Collapsible open={open}>
+        <Box pad={{ horizontal: 'small' }}>{children}</Box>
+        <Footer
+          pad={{
+            horizontal: 'xsmall',
+            vertical: 'small',
+          }}
+        >
+          <></>
```

Comment:
```suggestion
```
remove empty fragment?

Reply:
the empty fragment is serving the same purpose as `justify="end"` -- just using Footer's default justify between. But will change.

Reply:
done.